README.md

```diff
@@ -11,6 +11,24 @@
 
 <img src="plugin/src/main/webapp/img/logo-head.svg" width="192">
 
+
+<!--<a name="handling-secrets"></a>-->
+
+<!-- You can find more documentation about JCasC here: -->
+
+- [Handling Secrets](./docs/features/secrets.adoc)
+- [Exporting configurations](./docs/features/configExport.md)
+- [Validating configurations](./docs/features/jsonSchema.md)
+- [Triggering Configuration Reload](./docs/features/configurationReload.md)
+
+The configuration file format depends on the version of jenkins-core and installed plugins.
+Documentation is generated from a live instance, as well as a JSON schema you can use to validate configuration file
+with your favourite YAML tools.
+
+The JSON Schema documentation can be found [here](./docs/features/jsonSchema.md).
```

Comment:
Change json schema link to be added to the list of links above at the bottom and remove the sentence

Reply:
done